ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPartyManagementService.scala

```diff
@@ -218,7 +217,7 @@ private[apiserver] final class ApiPartyManagementService private (
             "update_mask",
           )
           displayNameO <- FieldValidations.optionalString(partyDetails.displayName)(Right(_))
-          partyRecord = ParticipantParty.PartyDetails(
+          partyRecord = domain.ParticipantPartyDetails(
```

Comment:
```suggestion
          partyRecord = ParticipantPartyDetails(
```

Reply:
Updating to use `ParticipantPartyDetails`